.changeset/changelog-config.js

```diff
@@ -1,3 +1,5 @@
+// Doesn't fully work but '.github/scripts/overwrite_changeset_changelog.py' forces the changelog to be formatted in GHA
```

Comment:
Should we just remove it if it doesn't work?

Reply:
it half works, so it's already giving me an easier format to overwrite than the standard `changeset` format from roo-node-types

Reply:
Can we remove the `getReleaseSummary` function then? That doesn't work right?

Reply:
removed 👍